sources/presentation/Xenko.Core.Presentation.Tests/Xenko.Core.Presentation.Tests.csproj

```diff
@@ -12,6 +12,10 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="xunit" Version="2.3.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
```

Comment:
It seems all the other references to `xunit.runner.visualstudio` don't have IncludeAssets/PrivateAssets.
Let's simplify those four lines to `<PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />`

Reply:
OK, will do.